.azuredevops/pipelineVariables/global.variables.yml

```diff
@@ -0,0 +1,84 @@
+variables:
+  ######################################
+  # Agent settings
+  ######################################
+
+  vmImage: 'ubuntu-latest' # Use this for Microsoft-hosted agents
+  poolName: '' # Use this for self-hosted agents
+  serviceConnection: 'CARML-Dev'
+
+  ######################################
+  # Source
+  ######################################
+
+  vstsOrganization: alsehr
```

Comment:
why is this hardcoded? Maybe better as a library variable?

Reply:
It is the org itself. So there is not really a need to have it in the variable. However, it's there temporary anyways as we have to agree eventually what org to use first